subsys/net/lib/lwm2m/lwm2m_rw_senml_json.c

```diff
@@ -291,10 +290,9 @@ static int put_begin(struct lwm2m_output_context *out, struct lwm2m_obj_path *pa
 	}
 
 	/*
-	 * Enable base name add if it is enabled
-	 * Base name is only added one time at first resource data
+	 * Init base level state for skip first object instance compare
```

Comment:
nit: it seems that this could be made a single-line comment now (`/* ... */`).

Reply:
I can change a comment to single line.